example/Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.5.1'
+gem 'rails', '~> 4.2'
 gem 'sqlite3'
 
 gem 'dotenv-rails'
-gem 'shopify_app', '~> 7.0.0'
+gem 'shopify_app', '~> 7.2.0'
```

Comment:
Keeping the example up to date with this repo isn't super well thought out. We kind of just update it as we remember. I had some sort of issue trying to specify the latest version but I don't recall exactly what it was.

It might be better to create a generator for the ESDK example code and remove this folder. It is kind of nice to have the example in the repo though but maybe if we did this then we could just re-generate this folder periodically.

Just some ideas.